src/docs/guides/laravel.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Deploy a Laravel App
+---
+
+Laravel is a PHP framework designed for web artisans who value simplicity and elegance in their code. It stands out for its clean and expressive syntax, and offers built-in tools to handle many common tasks found in modern web applications, making development smoother and more enjoyable.
+
+This guide covers how to deploy a Laravel app on Railway in three ways:
+
+1. [One-click deploy from a template](#one-click-deploy-from-a-template).
+2. [From a GitHub repository](#deploy-from-a-github-repo).
+3. [Using the CLI](#deploy-from-the-cli).
+
+## One-Click Deploy from a Template
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new/template/fWEWWf)
+
+This template sets up a basic Laravel application along with a Postgres database on Railway. You can also choose from a <a href="https://railway.app/templates?q=laravel" target="_blank">variety of Laravel app templates</a> created by the community.
+
+We highly recommend that [you eject from the template after deployment](/guides/deploy#eject-from-template-repository) to create a copy of the repo on your GitHub account.
+
+**Note:** If you see a "500 Server Error" on your deployed URL, it’s likely because Laravel is missing the **APP_KEY** or other required environment variables. To fix this, go to the Railway dashboard and add the necessary environment variables—especially the **APP_KEY**—and then redeploy the application. You can generate a new **APP_KEY** using this [Laravel APP KEY generator](https://generate-random.org/laravel-key-generator).
+
+## Deploy from a GitHub Repo
+
+To deploy a Laravel app on GitHub to Railway, follow the steps below:
+
+1. Create a <a href="https://railway.app/new" target="_blank">New Project.</a>
+2. Click **Deploy from GitHub repo**.
+3. Select your GitHub repo.
+    - Railway requires a valid GitHub account to be linked. If your Railway account isn't associated with one, you will be prompted to link it.
+4. Click **Add Variables**. 
+    - Add all your app environment variables.
+5. Click **Deploy**.
+
+Once the deployment is successful, a Railway [service](/guides/services) will be created for you. By default, this service will not be publicly accessible.
+
+**Note:** Railway will automatically detect that it's a Laravel app during [deploy and run your app via php-fpm and nginx](https://github.com/railwayapp/nixpacks/tree/main/src/providers/php). 
```

Comment:
It might be better to link to [the Nixpacks docs](https://nixpacks.com/docs/providers/php) here instead of linking to the source code.

Reply:
Yeah that makes sense. Thanks for the suggestion @aleksrutins